src/CMakeLists.txt

```diff
@@ -205,12 +205,6 @@ if (UCA_INCLUDE_DIRS AND UCA_LIBRARIES)
     link_directories(${UCA_LIBRARY_DIRS})
 endif ()
 
-if (GSL_FOUND)
-    include_directories(${GSL_INCLUDE_DIRS})
-    list(APPEND measure_aux_LIBS ${GSL_LIBRARIES})
-    list(APPEND ufofilter_SRCS ufo-measure-task.c)
-endif ()
```

Comment:
Could you also remove the `gsl_dep` check in `meson.build`?

Reply:
Done in 1fa1e295d1a41804ba4ca55daa7a92866162d092